src/Operators/Operators.jl

```diff
@@ -26,5 +26,6 @@ include("operator2stencil.jl")
 include("pointwisestencil.jl")
 include("remapping.jl")
 include("integrals.jl")
+include("arrayinterpolate.jl")
```

Comment:
Should we rename this file to `interpolate_array.jl` just so that the names match better?